internal/store/engine_state.go

```diff
@@ -65,7 +65,8 @@ type EngineState struct {
 	TiltfileState ManifestState
 
 	// from GitHub
-	LatestTiltBuild model.TiltBuild
+	LatestTiltBuild     model.TiltBuild
```

Comment:
nit: the above comment no longer applies to both of these lines
```suggestion
	LatestTiltBuild     model.TiltBuild // from GitHub
```